aerospike_helpers/cdt_ctx.py

```diff
@@ -83,6 +83,28 @@
 '''
 import aerospike
 
+#'index_type' - must be one of 'default', 'list', 'mapkeys', 'mapvalues'
+def index_type_string(index_type):
+    if index_type == aerospike.INDEX_TYPE_DEFAULT:
+        return "default"
+    if index_type == aerospike.INDEX_TYPE_LIST:
+        return "list"
+    if index_type == aerospike.INDEX_TYPE_MAPKEYS:
+        return "mapkeys"
+    if index_type == aerospike.INDEX_TYPE_MAPVALUES:
+        return "mapvalues"
+    return "invalid"
+
+#'index_datatype' - must be one of 'numeric', 'string', 'geo2dsphere'
+def index_datatype_string(index_datatype):
+    if index_datatype == aerospike.INDEX_NUMERIC:
+        return "numeric"
+    if index_datatype == aerospike.INDEX_STRING:
+        return "string"
+    if index_datatype == aerospike.INDEX_GEO2DSPHERE:
+        return "geo2dsphere"
+    return "invalid"
```

Comment:
These need docstrings so that sphinx can generate their documentation. If they are not intended for external use they should be private.